### DISCUSSION TITLE
mbp quickstart 失败 zsh: exec format error

### DISCUSSION THREAD

**Discussion by 7y-9** [alibaba/ilogtail#1047] [Help] [answered]
我尝试使用主页文档中标准的quickstart，但是启动失败了，貌似是我的环境问题？有没有大佬可以提供帮助
zsh: exec format error: ./ilogtail

mbp@mbp-Pro output % uname -a
Darwin mbp-Pro.local 22.5.0 Darwin Kernel Version 22.5.0: Thu Jun  8 22:22:23 PDT 2023; root:xnu-8796.121.3~7/RELEASE_ARM64_T6020 arm64
mbp@mbp-Pro output % make -version
GNU Make 3.81
Copyright (C) 2006  Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.

This program built for i386-apple-darwin11.3.0
mbp@mbp-Pro output % file ilogtail
ilogtail: ELF 64-bit LSB executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, BuildID[sha1]=e0f4b1d2ee946270396b22b7b75b152f8a8239bb, with debug_info, not stripped
mbp@mbp-Pro output % ./ilogtail 
zsh: exec format error: ./ilogtail

**Comment by 7y-9** [comment]
make 过程中没有报错而且docker镜像已经准备完成

**Reply by cyenxchen** [reply, marked as answer]
首先我记得官方是只适配了Linux和windows，其次你说了docker镜像已经准备好了，你就用docker启阿，为什么会直接裸跑ilogtail？
